cmd/build.go

```diff
@@ -25,7 +25,7 @@ const (
 )
 
 var (
-	configNew      = config.New
+	newConfigList  = config.NewConfigList
```

Comment:
I think just `New` is better.  
Now we feel below config, but if there are other settings at the root place, ConfigList sound not good. 
```
type Config struct {
	APIURL   string   `yaml:"api-url"`
	StoreURL string   `yaml:"store-url"`
	Token    string   `yaml:"token"`
	Launcher Launcher `yaml:"launcher"`
	filePath string   `yaml:"-"`
}
```

Reply:
Thx.
I renamed `config.NewConfigList` to `config.New`, `Config` to `Entry`, and `ConfigList` to `Config`.